src/components/vmd-appointment-card.component.scss

```
@@ -27,6 +27,9 @@
   &._mapharma {
     max-width: 10rem;
   }
+  &._avecmondoc {
+    max-width: 10rem;
```

J'aurais bien mis un `11rem` car le logo est pas super "fat" (et `11rem` est la plus grosse largeur qu'on puisse avoir sans commencer à engendrer des petits décalages de boutons)
```suggestion
    max-width: 11rem;
```